molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js

```diff
@@ -279,13 +279,13 @@
                     else{
                         info.setTitle("Chromosome:"+f.segment+" Position:"+ f.min);
                     }
-                }
 
-                var entityReportLink = $('<a href="javascript:void(0)">Show details</a>');
-                entityReportLink.click(function () {
-                	onRowInspect({id:f.id, name:getEntity().name});
-                });
-                info.add("", entityReportLink[0]);
+					var entityReportLink = $('<a href="javascript:void(0)">Show details</a>');
+					entityReportLink.click(function () {
+						onRowInspect({id:f.id, name:getEntity().name});
+					});
+					info.add("", entityReportLink[0]);
+				}
```

Comment:
diff ain't too clear: this code block was moved up a line to put it within "if(selectedTrack) {"